### DISCUSSION TITLE
umijs/max搭建的仓库在windows环境中git commit后huksy验证报错

### DISCUSSION THREAD

**Discussion by snowrain0722** [umijs/umi#11585] [Q&A]
![image](https://github.com/umijs/umi/assets/48642074/c0980bba-3a8f-4656-8a20-3a01343b73e3)
无法识别node命令，这个怎么解决？

**Comment by snowrain0722** [comment]
node.js在终端可以输出版本。

**Comment by fz6m** [comment]
先关闭 vscode 完全重启下 IDE 再看看，如果还不行就在网上找一下相关解决方案结合自身情况看下吧，比如 https://stackoverflow.com/questions/67063993/sh-husky-command-not-found

如果最终无法自己解决，就把 husky 的 `package.json#scripts.prepare` 和相关脚本注释掉先停用吧。

**Reply by snowrain0722** [reply]
搞定了，windows这么搞，https://zhuanlan.zhihu.com/p/637934871
